tls/s2n_crl.c

```diff
@@ -91,6 +91,47 @@ int s2n_crl_get_issuer_hash(struct s2n_crl *crl, uint64_t *hash)
     return S2N_SUCCESS;
 }
 
+int s2n_crl_validate_this_update(struct s2n_crl *crl)
```

Comment:
Are the names "this_update" and "next_update" standard for CRL, or did Openssl choose them? They seem kind of confusing, particularly "this_update", which seems to actually mean "last_update"?.

Maybe "validate_active" / "validate_ready_for_use" / "validate_last_updated_time" and "validate_not_expired" / "validate_expiration_time"? Or even just "validate_this_update_in_past" (or "validate_last_update_in_past") and "validate_next_update_in_future"?

Reply:
Yeah, the RFC calls them thisUpdate and nextUpdate. Openssl renamed thisUpdate to lastUpdate.

I like validate_active and validate_not_expired, so I changed the functions to this instead: a55d50f577c15af0c718fd0b0d75a7ecb5b9ca6d